service-common/rest-cdi/src/main/java/io/helidon/servicecommon/restcdi/HelidonRestCdiExtension.java

```diff
@@ -219,6 +225,10 @@ protected void recordProducerMethod(ProcessProducerMethod<?, ?> ppm) {
         recordProducerMember("recordProducerMethod", ppm.getAnnotatedProducerMethod(), ppm.getBean());
     }
 
+    /**
+     * Get producers.
```

Comment:
In general, the verb `get` should not be used in getter javadocs. So in this case it should be something relevant, e.g.

// I am not sure the actual text is correct, just to demonstrate how javadoc should be written
```
Map of bean instances to members producing them.
```